client/scripts/ui.js

```diff
@@ -204,6 +210,17 @@ class PeerUI {
     }
 }
 
+class QRCode {
+    constructor(slug) {
+        const img = document.createElement('img');
+        const url = new URL(slug, window.location.href);
+        const src = `http://api.qrserver.com/v1/create-qr-code/?color=000000&bgcolor=FFFFFF&data=${url.toString()}&qzone=0&margin=15&size=150x150&;ecc=L`;
```

Comment:
```suggestion
        const src = `https://api.qrserver.com/v1/create-qr-code/?color=000000&bgcolor=FFFFFF&data=${url.toString()}&qzone=0&margin=15&size=150x150&;ecc=L`;
```

Reply:
I also did https://github.com/SuperSandro2000/snapdrop/commit/be689e1165885bff7b36a816e7f7abbfda5e26b5 which generates the qr codes using javascript on your device rather than by using a remote service.